zeppelin-zengine/pom.xml

```diff
@@ -284,6 +284,11 @@
       <artifactId>commons-compress</artifactId>
       <version>${commons.compress.version}</version>
     </dependency>
+      <dependency>
```

Comment:
If you want to include this transitive dependency from the module `zeppelin-interpreter`, then this should be in the upper part of the `pom.xml`.